courses/static_analysis_suite/130_lab_message_review.rst

```diff
@@ -0,0 +1,160 @@
+**********************
+Lab - Message Review
+**********************
+
+..
+    Coding language
+
+.. role:: ada(code)
+    :language: Ada
+
+.. role:: C(code)
+    :language: C
+
+.. role:: cpp(code)
+    :language: C++
+
+..
+    Math symbols
+
+.. |rightarrow| replace:: :math:`\rightarrow`
+.. |forall| replace:: :math:`\forall`
+.. |exists| replace:: :math:`\exists`
+.. |equivalent| replace:: :math:`\iff`
+.. |le| replace:: :math:`\le`
+.. |ge| replace:: :math:`\ge`
+.. |lt| replace:: :math:`<`
+.. |gt| replace:: :math:`>`
+
+..
+    Miscellaneous symbols
+
+.. |checkmark| replace:: :math:`\checkmark`
+
+==============
+Lab Setup
+==============
+
+----------------------
+Perform Baseline Run
+----------------------
+
+* Use :toolname:`GNAT Studio` to open the :filename:`default.gpr` project in the :filename:`130_lab_message_review` folder
+* Start a baseline run:
+
+  * Select the menu :menu:`CodePeer` |rightarrow| :menu:`Analyze` in the menu bar at the top.
+  * Change :menu:`Analysis Level` to **4**
+  * Select the :menu:`Baseline run` switch
+  * Press :menu:`Execute`
+
+.. image:: codepeer/radar_analyze_level_4_baseline.png
+
+* Review the error message :file:`radar.adb:10`
+
+  * Select :menu:`Locations` tab in Message window
+  * Expand :file:`radar.adb` hierarchy (if not already)
+  * Select a message on line 10
+
+* The subprogram has two bugs:
```

Comment:
Suggest inserting "types" after "two" (for "two types of bugs")

Reply:
Fixed